pkg/chart/gcp.go

```diff
@@ -443,3 +443,55 @@ func roleBindingExistsInGCP(ctx context.Context, gcpProject, teamID, role string
 	roles := strings.Split(stdOut.String(), "\n")
 	return slices.Contains(roles, role), nil
 }
+
+func grantSATokenCreatorRole(ctx context.Context, teamID, gcpProject string) error {
+	role := "roles/iam.serviceAccountTokenCreator"
+
+	sa := fmt.Sprintf("%v@%v.iam.gserviceaccount.com", teamID, gcpProject)
+
+	cmd := exec.CommandContext(ctx,
+		"gcloud",
+		"iam",
+		"service-accounts",
+		"add-iam-policy-binding",
+		sa,
+		fmt.Sprintf("--role=%v", role),
+		fmt.Sprintf("--member=serviceAccount:%v", sa),
```

Comment:
Do we really need `--member` when we specify `sa` two lines above? Same question for the delete-function.

Reply:
Yes, we need to restrict the scope to the service account, not the project wild